pkg/kudoctl/packages/verifier/task/verify_basics.go

```diff
@@ -0,0 +1,29 @@
+package task
+
+import (
+	"github.com/kudobuilder/kudo/pkg/engine/task"
+
+	"github.com/kudobuilder/kudo/pkg/kudoctl/packages"
+	"github.com/kudobuilder/kudo/pkg/kudoctl/packages/verifier"
+)
+
+// ReferenceVerifier verifies tasks producing errors for tasks referenced in plans that do not exist and warnings for tasks which are not used in a plan
+type BasicVerifier struct{}
+
+func (BasicVerifier) Verify(pf *packages.Files) verifier.Result {
+	res := verifier.NewResult()
+	res.Merge(tasksWellDefined(pf))
+	return res
+}
+
+func tasksWellDefined(pf *packages.Files) verifier.Result {
+	result := verifier.NewResult()
+	for _, tt := range pf.Operator.Tasks {
+		tt := tt
+
+		if _, err := task.Build(&tt); err != nil {
```

Comment:
I like this verifier! I put the validation code in the `task.go` originally because we haven't had a good place for it back then. Wdyt about little refactoring: move that code and corresponding tests here?

Reply:
Refactoring might be possible. I tried it a little bit yesterday, but couldn't find a good structure, so I'm leaving it as is for now.